src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -526,6 +526,7 @@ void CascadiaSettings::_AppendWslProfiles(std::vector<TerminalApp::Profile>& pro
             std::getline(wlinestream, distName, L' ');
             auto WSLDistro{ _CreateDefaultProfile(distName) };
             WSLDistro.SetCommandline(L"wsl.exe -d " + distName);
+            WSLDistro.SetStartingDirectory(L"~");

```

Comment:
alas, this will almost certainly not work properly. we'll pass it to `CreateProcess`, which doesn't understand linux paths.

Reply:
Yeah, it has the workaround to set it to `DEFAULT_STARTINGDIRECTORY` right now, and I figured it would make it future proof once something happens with #592.  I can change it back if you like, but #592 also seems like some low hanging fruit that I could work on after this.

Reply:
No comments on this for 2 days, so I'll remove that code.